src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -135,6 +135,14 @@ const routes: Routes = [
           resource: 'api.cluster.user@1.0'
         }
       },
+      {
+        path: 'cluster/user/edit',
+        component: CrudFormComponent,
+        data: {
+          breadcrumbs: 'Cluster/Users',
+          resource: 'api.cluster.user@1.0'
+        }
+      },
```

Comment:
actually we should fix this. the `create` and `edit` or whatever actions comes after should be a child of the `users` path.
```suggestion
            {
        path: 'users',
        data: {
          breadcrumbs: 'Cluster/Users',
          resource: 'api.cluster.user@1.0'
        },
        children: [
          {
            path: '',
            component: CRUDTableComponent
          },
          {
            path: URLVerbs.CREATE,
            component: CrudFormComponent,
            data: {
              breadcrumbs: URLVerbs.CREATE,
            }
          },
          {
            path: URLVerbs.EDIT,
            component: CrudFormComponent,
            data: {
              breadcrumbs: URLVerbs.EDIT,
            }
          }
        ]
      },
```